frontend/src/router.js

```diff
@@ -25,10 +25,10 @@ let store = createStore(fetchData, applyMiddleware(thunk));
 const router = (
   <Provider store={store}>
     <Router>
-      <div className="site container-fluid">
+      <div className="site ">
```

Comment:
Why did we remove the container classes here? We probably didn't need both, but we definitely need at least one.